tools/autograd/gen_variable_type.py

```diff
@@ -887,16 +862,12 @@ def unpack_args(f: NativeFunction) -> Tuple[List[str], List[Binding]]:
     body: List[str] = []
     unpacked_bindings: List[Binding] = []
 
-    if f.use_c10_dispatcher.dispatcher_uses_new_style():
-        bindings = [r for a in f.func.schema_order_arguments()
-                    for r in cpp.argument(a,
-                                          method=False,
-                                          cpp_no_default_args=set(),
-                                          faithful=False,
-                                          has_tensor_options=False)]
-    else:
-        sig_group = CppSignatureGroup.from_native_function(f, method=False)
-        bindings = list(sig_group.signature.arguments())
+    bindings = [r for a in f.func.schema_order_arguments()
+                for r in cpp.argument(a,
+                                        method=False,
```

Comment:
total nit but if you're in here again before landing, it'd be nice to indent these in the conventional way